.github/workflows/ui-build.yml

```diff
@@ -0,0 +1,26 @@
+name: Build & Test UI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-test:
+    name: Build & Test UI
+    runs-on: ubuntu-latest
```

Comment:
can you change runs-on to be similar to https://github.com/PeerDB-io/peerdb/pull/383/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR11 for both this and the other file?

For context, we are testing out ubicloud runners for lower price and better perf.